py-pkg/setup.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+from setuptools import setup
+
+INSTALL_REQUIREMENTS_FILE = Path(__file__).parent / "requirements.txt"
+
+
+def load_requirements(path: Path):
+    requirements = []
+    with path.open("r") as fp:
+        for line in fp.readlines():
+            if line.startswith("-r"):
+                requirements += load_requirements(line.split(" ")[1].strip())
+            else:
+                requirement = line.strip()
+                if requirement and not requirement.startswith("#"):
+                    requirements.append(requirement)
+    return requirements
+
+
+setup(
+    name="pkgnet",
+    version="0.1",
+    description="",
```

Comment:
I think you should consider reading in `py-pkg/README.md` and setting that as the description, like this: https://github.com/jameslamb/doppel-cli/blob/3a720b0902087d498b3f614245e5e9a5d8be87bb/setup.py#L21-L22

If you do that plus `long_description_content_type="text/markdown",`, it'll become the homepage on PyPi without us writing any new documentation.